src/ratelimit_manager.cpp

```diff
@@ -165,18 +168,17 @@ const std::vector<rate_limit_rule_t> RateLimitManager::get_all_rules() {
 
 const std::vector<rate_limit_status_t> RateLimitManager::get_banned_entities(const RateLimitedEntityType entity_type) {
     std::shared_lock<std::shared_mutex> lock(rate_limit_mutex);
```

Comment:
Existing convention is `std::shared_lock lock(rate_limit_mutex);` rather than `std::shared_lock<std::shared_mutex> lock(rate_limit_mutex)`. (less verbose also). Check other instances of this pattern to fix as well.